client/packages/common/src/intl/locales/en/common.json

```diff
@@ -476,6 +476,7 @@
   "heading.notification-preferences": "Notification Preferences",
   "heading.order": "Order",
   "heading.order-info": "Order info",
+  "heading.other": "Other",
```

Comment:
On first reading I found 'Other' as a category pretty jarring. Wonder if this should be something else?

ie if this is for custom reports for specific clients, it could be part of dialect translations for that client for a custom heading dependent on what reports they are using.

Reply:
mSupply also has an other category too